src/components/common/Footer/DesktopFooter.js

```diff
@@ -14,27 +14,35 @@ const { Title } = Typography;
 function DesktopFooter() {
   return (
     <>
-      <Row
-        justify="space-between"
-        style={{ maxWidth: '1100px', margin: '0 auto' }}
-      >
+      <Row justify="space-between">
         <Col>
           <Title
             className="title"
             style={{ color: '#ffffff', fontWeight: '400' }}
             level={3}
           >
-            Contact
```

Comment:
Thank you for moving these around! They make way more sense in the new layout.

Reply:
And looks infinitely better without the bullet points!